recipes/xalan-c/build.sh

```diff
@@ -0,0 +1,19 @@
+
+export XERCESCROOT=${PREFIX}
+export XALANCROOT=$(pwd)
+
+if [ $(uname) == Darwin ]; then
+    platform=macosx
+    CC=clang
+    CXX=clang++
+    EXTRA_CXX_OPTIONS="-z -stdlib=libc++"
+else
+    platform=linux
+    CC=gcc
+    CXX=g++
+fi
+
```

Comment:
No need to set CC/CXX . They will be automatically set. EXTRA_CXX_OPTIONS is probably also not needed here.